widget_driver/doc/drivers_without_generation.md

```diff
@@ -0,0 +1,60 @@
+# Drivers Without Generation
+
+You do not need to use code generation to use `WidgetDrivers`. You can just as well create all the needed code manually. It is just a bit more manual work for you 😅
+
+It is actually not so much code you have to create for each `driver`. But as you project grows it still adds up.
+
+*So the recommendation would still be to use code generation.*
+
+## What do I need to do
+
+You just need two extra classes!  
+Except for the actual `WidgetDriver` you need to create a `TestDriver` version of your `Driver`. And you need to create a `WidgetDriverProvider` which can create an instance for your `driver` and `testDriver`.
+
+### Creating the TestDriver
+
+The `TestDriver` is used when your widget is created during widgetTests. It cannot have any dependencies to any instance of something. It should just contain hard coded values. *(It can contain static values from some other type.)*
```

Comment:
```suggestion
The `TestDriver` is used when your widget is created during widget tests. It can't have any dependencies to any instance of something. It should just contain hard coded values. *(It can contain static values from some other type.)*
```